content/FIDO2/FIDO2_WebAuthn_Developer_Guide/WebAuthn_Client_Authentication.adoc

```diff
@@ -70,7 +78,11 @@ https://www.w3.org/TR/webauthn/images/webauthn-authentication-flow-01.svg[W3C We
 
 **clientExtensionResults():** method returns values for zero or more WebAuthn extensions.
 
-==== Parsed authenticatorData Example:
+==== Example
+
+**Parsed authenticatorData**
+
+_Note: In the WebAuthn API, the `authenticatorData` is actually an `ArrayBuffer`. This example displays it in Base64 for readability_
```

Comment:
I don't think this note is necessary; it should be clear enough that this is a conceptual deconstruction of the byte array.

Reply:
Thanks! Removed in ae46aac